game-core/src/main/java/games/strategy/engine/data/RouteFinder.java

```diff
@@ -12,18 +13,32 @@
 // TODO this class doesn't take movementcost into account... typically the shortest route is the fastest route, but not
 // always...
 class RouteFinder {
+
   private final GameMap map;
   private final Predicate<Territory> condition;
   private final Map<Territory, Territory> previous;
+  private final Collection<Unit> units;
+  private final PlayerID player;
 
   RouteFinder(final GameMap map, final Predicate<Territory> condition) {
     this.map = map;
     this.condition = condition;
     previous = new HashMap<>();
+    units = new HashSet<>();
+    player = null;
+  }
```

Comment:
With the addition of the second constructor, can the body of this constructor simply be the following?

```java
this(map, condition, new HashSet<>(), null);
```

Reply:
Done.